package/src/components/MessageInput/FileUploadPreview.tsx

```diff
@@ -113,72 +117,74 @@ const FileUploadPreviewWithContext = <
     },
   } = useTheme();
   const renderItem = ({ index, item }: { index: number; item: FileUpload }) => (
-    <UploadProgressIndicator
-      action={() => {
-        uploadFile({ newFile: item });
-      }}
-      active={item.state !== FileState.UPLOADED && item.state !== FileState.FINISHED}
-      style={styles.overlay}
-      type={
-        item.state === FileState.UPLOADING
-          ? ProgressIndicatorTypes.IN_PROGRESS
-          : item.state === FileState.UPLOAD_FAILED
-          ? ProgressIndicatorTypes.RETRY
-          : undefined
-      }
-    >
-      <View
-        style={[
-          styles.fileContainer,
-          index === fileUploads.length - 1
-            ? {
-                marginBottom: 0,
-              }
-            : {},
-          {
-            borderColor: grey_whisper,
-            width: flatListWidth - 16,
-          },
-          fileContainer,
-        ]}
+    <>
+      <UploadProgressIndicator
+        action={() => {
+          uploadFile({ newFile: item });
+        }}
+        active={item.state !== FileState.UPLOADED && item.state !== FileState.FINISHED}
+        style={styles.overlay}
+        type={
+          item.state === FileState.UPLOADING
```

Comment:
Just a personal preference thing, but might it be a bit easier to read this if the nested ternary were to be extracted into a variable or something? Or maybe even if it's just wrapped in parentheses?

Reply:
I realize this wasn't introduced in this PR - don't mind me for the sake of getting this merged :)